app/assets/javascripts/discourse/app/components/image-uploader.js

```diff
@@ -82,9 +74,19 @@ export default Component.extend(UploadMixin, {
   },
 
   _openLightbox() {
-    next(() =>
-      $(this.element.querySelector("a.lightbox")).magnificPopup("open")
-    );
+    next(() => {
```

Comment:
why next and not afterRender ?

Reply:
I tested this a bit and I don't think it's needed so I removed it.